src/App.js

```diff
@@ -1,12 +1,20 @@
 import './App.css';
```

Comment:
First external imports should occur then 1 line space, then internal files and component imports and then 1 line space and finally css imports